website/content/docs/k8s/upgrade/index.mdx

```diff
@@ -115,112 +120,37 @@ to update to the new version. Before you upgrade to a new version:
-  ```
-
-1. Take specific note if `consul-server, StatefulSet` is listed, as it means your Consul server statefulset will be redeployed.
-
-  If your Consul server statefulset needs to be redeployed, follow the same pattern for upgrades as
-  on other platforms by redeploying servers one by one. Refer tp [Upgrading Consul](/consul/docs/upgrading) for more information.
-
-  If neither the server statefulset is not being redeployed,
-  then you can continue with the Helm upgrade without any specific sequence to follow.
-
-## Upgrade Consul servers
-
-Initiate the server upgrade:
-
-1. Change the `global.image` value to the desired Consul version.
-1. Set the `server.updatePartition` value to the number of server replicas. By default, there are 3 servers, so you would set this value to `3`.
-1. Set the `updateStrategy` for clients to `OnDelete`.
-
-  <CodeBlockConfig filename="values.yaml">
-
-  ```yaml
-  global:
-    image: 'consul:123.456'
-  server:
-    updatePartition: 3
-  ```
-
-  </CodeBlockConfig>
-
-  The `updatePartition` value controls how many instances of the server
-  cluster are updated. Only instances with an index _greater than_ the
-  `updatePartition` value are updated (zero-indexed). Therefore, by setting
-  it equal to replicas, updates should not occur immediately.
-
-1. Next, perform the upgrade:
-
-  ```shell-session
-  $ helm upgrade consul hashicorp/consul --namespace consul --version <your-version> --values /path/to/your/values.yaml
-  ```
-
-  This command does not cause the servers to redeploy, although the resource is updated. If
-  everything is stable, decrease the `updatePartition` value by one
-  and performing `helm upgrade` again. This will cause the first Consul server
-  to be stopped and restarted with the new image.
+~> Note: If you don't pass the `--version` flag when upgrading a Helm chart, Helm uses the most up-to-date version of the chart in its local cache, which may result in an unintended version upgrade.
 
-1. Wait until the Consul server cluster is healthy again (30s to a few minutes).
-   This can be confirmed by issuing `consul members` on one of the previous servers,
-   and ensuring that all servers are listed and are `alive`.
+## Consul server restarts/upgrades
 
-1. Decrease `updatePartition` by one and upgrade again. Continue until
-   `updatePartition` is `0`. At this point, you may remove the
-   `updatePartition` configuration. Your server upgrade is complete.
+-> *Note for older versions*: For versions of Consul on Kubernetes prior to `1.4.0`, we recommended using the `server.updatePartition` setting to slowly upgrade
+Consul servers. This is no longer necessary due to some changes we made to increase the stability of upgrades, but if you
+are upgrading a version of the chart older than `1.4.0`, use the version drop-down at the top of the page to select a version
+<= `1.17.0` (corresponding to the Consul version in your chart, not the chart version) which contains those instructions.
```

Comment:
```suggestion
Consul servers. Refer to an older version of the documentation for instructions on upgrading to a version of the chart older than `v1.4.0`. Use the version drop-down at the top of this page to select a version older than or equal to `v1.17.0`. Consul documentation versions correspond to the Consul version in your chart, not the chart version, that contains the instructions.
```

Per the style guide, avoid using parentheses and mathematical expressions in prose.